src/lib/DiskTab.svelte

```diff
@@ -8,15 +8,22 @@
 	{
 		if(state == "START")
 			state = "CONFIRM";
-		else
-			dispatch('reset');
+		else if (state == "CONFIRM") {
+			state = "RESETTING";
+			setTimeout(() => {
+				dispatch('reset');
+				state = "START";
```

Comment:
I don't think we should reset the state after a timeout. After the reset process is complete the page will reload. The purpose here is to make sure the user is aware that reset is in progress and might take a bit.